src/praisonai-agents/tests/unit/runtime/test_registry.py

```
@@ -0,0 +1,201 @@
+    assert runtime.supports("gpt-4") is True
+    
+    # Test run_turn (may fail if no OpenAI key, but should not crash)
+    try:
+        result = await runtime.run_turn("Hello test")
+        assert isinstance(result, RuntimeResult)
+        # Content might be empty if no API key, but should be string
+        assert isinstance(result.content, str)
+        assert isinstance(result.metadata, dict)
+    except Exception:
+        # It's OK if this fails due to missing API keys in tests
+        pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)